cli/command/service.go

```diff
@@ -10,6 +15,261 @@ func NewServiceCommand(b *CommandBase) *ServiceCommand {
 	return &ServiceCommand{b}
 }
 
-func (e *ServiceCommand) Command() cli.Command {
-	return cli.Command{}
+func (s *ServiceCommand) Command() cli.Command {
+	return cli.Command{
+		Name:  "service",
+		Usage: "manage Layer0 services",
+		Subcommands: []cli.Command{
+			{
+				Name:      "create",
+				Usage:     "create a new service",
+				Action:    s.create,
+				ArgsUsage: "ENVIRONMENT NAME DEPLOY",
+				Flags: []cli.Flag{
+					cli.StringFlag{
+						Name:  "loadbalancer",
+						Usage: "attach the service to the specified load balancer",
+					},
+					cli.BoolFlag{
+						Name:  "nowait",
+						Usage: "don't wait for deployment to complete before returning",
+					},
+				},
+			},
+			{
+				Name:      "delete",
+				Usage:     "delete a service",
+				Action:    s.delete,
+				ArgsUsage: "NAME",
+				Flags: []cli.Flag{
+					cli.BoolFlag{
+						Name:  "nowait",
+						Usage: "don't wait for the job to complete before returning",
+					},
+				},
+			},
+			{
+				Name:   "list",
+				Usage:  "list all services",
+				Action: s.list,
+			},
+			{
+				Name:      "logs",
+				Usage:     "get the logs for a service",
+				Action:    s.logs,
+				ArgsUsage: "NAME",
+				Flags: []cli.Flag{
+					cli.IntFlag{
+						Name:  "tail",
+						Usage: "number of lines from the end to return",
+						Value: 0,
```

Comment:
I don't think you need to set this if it's already 0

Reply:
Removed. ✅